localstripe/resources.py

```diff
@@ -3852,7 +3858,10 @@ def _api_approve(cls, id: str, amount=None, metadata=None, **kwargs):
             logger.warning('Unexpected ' + ', '.join(kwargs.keys()))
             raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
 
-        amount = try_convert_to_int(amount)
+        if amount is not None:
+            amount = try_convert_to_int(amount)
```

Comment:
try_convert_to_int already does this though?

Reply:
I am not familiar with Python but this would try convert value to int else it is throwing exception.

https://stackoverflow.com/questions/8075877/converting-string-to-int-using-try-except-in-python

Reply:
We already do it here 
https://github.com/quadpay/localstripe/blob/b44d27a16fa9ebe61e59d5df08be955cb6e24c1e/localstripe/utilities.py#L24

Reply:
Also, can you undo other changes for non-dispute changes